docs/rewind/intro.mdx

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 1
+---
+# Rewind Download and Support
+
+Welcome to the Fourier Rewind support site. Here you can download the latest version of Fourier Rewind,
+peruse the FAQs or take a read of the manual.
+
+## Download Rewind
+
+The latest version of Fourier Rewind can be downloaded by clicking here:
```

Comment:
To avoid confusion or hesitation, most websites mention the target platform right next to the download link.

Consider: `can be downloaded for Mac by clicking here`

Reply:
Done :)